pkg/storage/dimension/dimension.go

```diff
@@ -6,21 +6,21 @@ import (
 	"sync"
 )
 
-type key []byte
```

Comment:
why make this one public?

Reply:
`Intersection()` is a public method which returns an array of items of `Key` type which itself is not public.